cmd/cli/multiregion/dns.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright © 2023 SIL International
+*/
+
+package multiregion
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type DnsCommand struct {
+	cfClient   *cloudflare.API
+	cfZone     *cloudflare.ResourceContainer
+	domainName string
+	testMode   bool
+}
+
+func InitDnsCmd(parentCmd *cobra.Command) {
+	parentCmd.AddCommand(&cobra.Command{
+		Use:   "dns",
+		Short: "DNS Failover and Failback",
+		Long:  `Configure DNS CNAME values for primary or secondary region hostnames`,
+		Run: func(cmd *cobra.Command, args []string) {
+			runDnsCommand()
+		},
+	})
+}
+
+func runDnsCommand() {
+	pFlags := getPersistentFlags()
+
+	f := newDnsCommand(pFlags)
+
+	f.setDnsToSecondary()
+}
+
+func newDnsCommand(pFlags PersistentFlags) *DnsCommand {
+	d := DnsCommand{
+		testMode:   pFlags.readOnlyMode,
+		domainName: viper.GetString("domain-name"),
+	}
+
+	if d.domainName == "" {
+		log.Fatalf("Cloudflare Domain Name is not configured. Use 'domain-name' parameter.")
+	}
+
+	cfToken := viper.GetString("cloudflare-token")
+	if cfToken == "" {
+		log.Fatalf("Cloudflare Token is not configured. Use 'cloudflare-token' parameter.")
+	}
+
+	api, err := cloudflare.NewWithAPIToken(cfToken)
+	if err != nil {
+		log.Fatal("failed to initialize the Cloudflare API:", err)
+	}
+	d.cfClient = api
+
+	zoneID, err := d.cfClient.ZoneIDByName(d.domainName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("Using domain name %s with ID %s\n", d.domainName, zoneID)
+	d.cfZone = cloudflare.ZoneIdentifier(zoneID)
+
+	return &d
+}
+
+func (d *DnsCommand) setDnsToSecondary() {
+	fmt.Println("Setting DNS records to secondary...")
+
+	mfaApiName := getOption("mfa-api-name", "mfa-api")
```

Comment:
It might be good to include a comment on why we name things `mfa` and `twosv`, which are basically synonyms. 
IIrc, `mfa` is what we're using in AWS for the TOTP endpoints and `twosv` is what we're using for Webauthn endpoints.

It's a shame we didn't distinguish them more intuitively.  Maybe one day the mfa stuff will be merged into the twosv stuff.